bang/magicollections.py

```diff
@@ -1943,24 +1948,20 @@ def to_fields(self, item, *args, **kwargs):
             fields = super(SongCollection.ItemView, self).to_fields(item, *args, **kwargs)
             note_image = u'{}img/note.png'.format(RAW_CONTEXT['static_url'])
             for difficulty, verbose_name in models.Song.DIFFICULTIES:
-                notes = getattr(item, u'{}_notes'.format(difficulty), None)
-                difficulty_level = getattr(item, u'{}_difficulty'.format(difficulty), None)
-                if notes or difficulty_level:
+                diff_value=''
+                diff = getattr(item, u'{}_difficulty'.format(difficulty), None)
+                if diff != None: diff_value=mark_safe(u'{big_images}{small_images}'.format(diff=diff,
```

Comment:
`if diff is not None`
\+ move what's after the `:` to the next line.
\+ add spaces around `=`
\+ use function previously created to generate images.

Reply:
✅